flows/actions/testdata/call_resthook.json

```diff
@@ -41,11 +41,7 @@
                         "created_on": "2018-10-18T14:20:30.000123Z",
                         "text": "Hi everybody",
                         "type": "msg",
-                        "urn": {
-                            "display": "(206) 555-1212",
-                            "path": "+12065551212",
-                            "scheme": "tel"
-                        },
+                        "urn": "tel:+12065551212",
```

Comment:
This is a change to the payload we're sending our resthooks... not sure what consequences that would have? @nicpottier

Reply:
Not OK! Needs to be like it was before.

Reply:
PS oddly `contact > urn` in the payload has always been a string and `input > urn` was a dict

Reply:
Ok I've updated `DefaultWebhookPayload` expression - it's impressively complex https://github.com/nyaruka/goflow/pull/590/files#diff-15cdc0e10766d2199f1f84144133c57cR23

Reply:
Ya this is where having a `DICT` function or literal would be awful nice.